commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/PurePursuitControllers.scala

```diff
@@ -80,99 +81,98 @@ trait PurePursuitControllers {
   /**
     * returns an angle limited from -180 to 180 equivalent to the input
     * @param degrees
-    * @return angle betwee -270 and 90
+    * @return angle betwee -90 and 90
```

Comment:
I deal with the case that the target angle is 180 or similar by calling this method. You are write that it is not returning an equivalent angle, but that's not the purpose of this method